cspell/.cspell.json

```
@@ -46,11 +46,16 @@
         "**/*.caffemodel",
         "**/*.cu",
         "**/*.html",
+        "**/*.jpeg",
+        "**/*.jpg",
+        "**/*.mp3",
```

```suggestion
        "**/*.m4a",
        "**/*.mp3",
```
cSpell should ignore `jpeg` and `jpg` by default. So I'd like to delete the lines.
Instead, ignoring `m4a` might be helpful.

---

Okay, thanks for info!